mock/py/mockbuild/scm.py

```diff
@@ -147,6 +148,11 @@ def prepare_sources(self):
             sys.exit(5)
         self.spec = sf
 
+        # Use specified sources directory
+        sources_dir = self.src_dir
+        if self.int_src_dir:
+            sources_dir += "/" + self.int_src_dir
```

Comment:
Can you use `sources_dir = os.path.join(sources_dir, self.int_src_dir)`?

Reply:
Fixed (and accidentally rebased).